DS/DSAssignment2/InfixToPostfix.java

```diff
@@ -0,0 +1,79 @@
+
+public class InfixToPostfix {
+
+	private Stack theStack;
+	private String input;
+	private String output = "";
+
+	public InfixToPostfix(String in) {
+		input = in;
+		int stackSize = input.length();
+		theStack = new Stack(stackSize);
+	}
+
+	public String doTrans() {
+		for (int j = 0; j < input.length(); j++) {
+			char ch = input.charAt(j);
+			switch (ch) {
+			case '+':
+			case '-':
+				gotOper(ch, 1);
+				break;
+			case '*':
+			case '/':
+				gotOper(ch, 2);
+				break;
+			case '(':
+				theStack.push(ch);
+				break;
+			case ')':
+				gotParen(ch);
+				break;
+			default:
+				output = output + ch;
+				break;
+			}
+		}
+		while (!(theStack.getTop() == -1)) {
+			output = output + theStack.peep();
+			theStack.pop();
+		}
+		System.out.println(output);
+		return output;
+	}
+
+	public void gotOper(char opThis, int prec1) {
```

Comment:
ACTION: 'gotOper()' and 'gotParen()' should be 'private' as they are not being used from outside (your test methods).